packages/orchestrator-ui-components/src/pages/metadata/WfoWorkflowsPage.tsx

```diff
@@ -128,6 +129,13 @@ export const WfoWorkflowsPage = () => {
             columnType: ColumnType.DATA,
             label: t('description'),
             width: '450px',
+            renderData: (value, row) =>
```

Comment:
Looks like code duplication from WfoTasksPage, but on closer inspection almost this whole function/page seems to be duplicated. Maybe factor out some common code?

Reply:
Refactoring this page should be addressed in a separate ticket. The added lines are sufficiently different.